packages/javascript/src/interfaces/options.ts

```diff
@@ -7,6 +7,7 @@ export interface AppsignalOptions extends BaseOptions {
   namespace?: string
   revision?: string
   ignoreErrors?: RegExp[]
+  matchPath?: RegExp | RegExp[]
```

Comment:
When in doubt, be more explicit?

```suggestion
  matchBacktraceLinePath?: RegExp | RegExp[]
```

Reply:
Agreed! I think we can drop the `Line` though -- `matchBacktracePath` (or `matchBacktracePaths`?) will be specific enough.

Reply:
Let's go for `matchBacktracePaths` then.